examples/multi_nic_common/example.tfvars

```diff
@@ -171,7 +171,7 @@ routes = {
 
 vmseries_common = {
   ssh_keys            = "admin:<YOUR_SSH_KEY>"
-  vmseries_image      = "vmseries-flex-byol-1029h1"
+  vmseries_image      = "vmseries-flex-byol-10210h9"
```

Comment:
Maybe we can use the same vmseries image across the board ? Like the latest recommended 11 image with the CVE hotfix.

Reply:
`10.2.10-h9` is currently the preferred image for 10.2.

Reply:
I know - but what I am suggesting is to have the same version across all images - which is 11.X.